lib/pymedphys/_vendor/patchlogging.py

```diff
@@ -46,7 +46,7 @@ def apply_logging_patch():
         )
 
     if _patch_applied:
-        raise ValueError("This patch has already been applied.")
```

Comment:
This reverses behaviour and I will assume that this is intentional but I thought I would mark it just in case

Reply:
Yup good call marking it :) yup it was intentional. I think I was overzealous on the first pass. It didn't need to raise an error, instead just not re-apply the patch.